include/SDL3/SDL_audio.h

```diff
@@ -80,15 +80,17 @@ typedef Uint16 SDL_AudioFormat;
 /* @{ */
 
 #define SDL_AUDIO_MASK_BITSIZE       (0xFF)
-#define SDL_AUDIO_MASK_DATATYPE      (1<<8)
-#define SDL_AUDIO_MASK_ENDIAN        (1<<12)
+#define SDL_AUDIO_MASK_FLOAT         (1<<8)
+#define SDL_AUDIO_MASK_LIL_ENDIAN    (1<<12)
+#define SDL_AUDIO_MASK_BIG_ENDIAN    (1<<13)
```

Comment:
It feels a bit strange having separate bits for lil/big when they should always be mutually exclusive.

Reply:
Yeah, I kind of agree, but then I found the code in SDL_audiocvt.c that strips off the mask to find the endian independent format.

Reply:
I suppose so, but I'd say that's more of an edge case that isn't worth changing the format for.

Also, the old MSB formats now equal the new LE formats, which is quite confusing.

Reply:
Okay, I'll change that part back, thanks for the feedback!